### ISSUE TITLE
Is there an s3 support for this ?

### ISSUE THREAD

**Issue by pamio** [khrykin/sequelize-file#2]
Can we upload images to s3 just like `paperclip` and `carrierwave` `gems` ? Is that included in roadmap ?

**Comment by khrykin** [comment]
@pamio no, by now all file saving logic is hard coded to be dealing with local filesystem, but sure I'm planning to extract it to make it possible to add more options - s3, dropbox, whatever. I just don't personally need this yet, so PRs are welcome
